src/helpers/parse_snak.ts

```diff
@@ -105,22 +108,16 @@ for (const [ datatype, parser ] of Object.entries(parsers)) {
   normalizedParsers[normalizeDatatype(datatype)] = parser
 }
 
-export function parseClaim (datatype, datavalue, options, claimId) {
-  // Known case of missing datatype: form.claims, sense.claims, mediainfo.statements
+export function parseSnak (datatype: DataType | void, datavalue: SnakValue, options: SimplifySnakOptions) {
```

Comment:
datatype can be void? `void` should only be a return type. Is it `datatype: Datatype | undefined`?

Reply:
done in 740a737